qiskit_dynamics/models/operator_collections.py

```diff
@@ -934,13 +743,13 @@ def evaluate_rhs(
             # A matrix
             if self._static_dissipators is None:
                 dissipators_matrix = np.sum(
-                    -0.5 * dis_sig_vals * self._dissipator_products, axis=-1
+                    -0.5 * dis_coefficients * self._dissipator_products, axis=-1
```

Comment:
For the dissipator terms, there is a `-0.5` associated with `_static_dissipators_product_sum` and `_dissipator_products`, which is the `-1/2` in the [Lindblad dissipator sum](https://en.wikipedia.org/wiki/Lindbladian). When making this PR I noticed that the `-0.5` was being multiplied on the `dis_coefficients` every time `evaluate_rhs` was called, rather than being folded directly into `_dissipator_products`. So I just moved the `-0.5` into the definition of `_dissipator_products`, which is how it should have been from the beginning.